config/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.7"
+services:
+  db:
+    image: "mysql:5.7"
+    restart: always
+    environment:
+     MYSQL_ROOT_PASSWORD: root
+     MYSQL_USERNAME: root
+     MYSQL_PASSWORD: root
+        - '3306:3306'
+    expose:
+      - '3306'
+    volumes:
+      - db:/var/run/mysqld
+    logging:
+            driver: none
+    ports:
+      - "33061:3306"
```

Comment:
Do you need to document port #s in the normal documentation?

Reply:
Based on my understanding, I think we don't need to because Port 3306 is usually used as the default port for the MySQL protocol.